src/components/ec-blog-item/ec-blog-item.story.js

```diff
@@ -0,0 +1,32 @@
+import EcBlogItem from './ec-blog-item.vue';
+
+export default {
+  title: 'Blog Item',
+  component: EcBlogItem,
+};
+
+const Template = (args, { argTypes }) => ({
+  props: Object.keys(argTypes),
+  components: {
+    EcBlogItem,
+  },
+  template: `
+      <div class=" ec-blog-item tw-grid tw-border-b tw-border-solid tw-border-gray-6" >
```

Comment:
there is an extra space before ec-blog-item `class=" ec-blog-item`

Reply:
Probably the utility classes could go under the BEM class below